src/download.js

```diff
@@ -1,13 +1,6 @@
-var zip = require("./zip");
+var zip = require('./zip');
+var saveAs = require("file-saver").saveAs;
 
-module.exports = async function (gj, options) {
-  var content = await zip(gj, options);
-  var link = document.createElement("a");
-  link.href = "data:application/zip;base64," + content;
-  link.rel = "noopener";
-  if ((options && options.filename) || options.folder) {
-    link.download = (options.filename || options.folder) + ".zip";
-  }
-  link.click();
-  window.URL.revokeObjectURL(link.href);
+module.exports = function(gj, options) {
+    zip(gj, options).then(function(blob) { saveAs(blob, options.file + '.zip'); });
```

Comment:
where is `options.file` being defined?

Reply:
folder ?? filename ?? <default>

Reply:
Good catch, I've updated this to correctly use the filename/folder options in the interface. It will default to the filename for the zip file name (so `${options.filename}.zip` otherwise it will use the folder name `${options.folder}.zip`. What do you think about that? That seems to best reflect the original logic

Reply:
Seems to resolve the issue for me. Thanks!